domains/akborana.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "akborana3",
+           "email": "akayonfire3@gmail.com",
+           "discord": "1042457884736368721"
+        },
+    
+        "record": {
+            "CNAME": "akborana3.github.io"
```

Comment:
404 Site not found

Reply:
He has a site on https://akborana3.github.io/Portfolio/ so it's ok, when he adds the domain to that repo it'll work as normal